src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-ProfileCard/Components/ProfileCard.jsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable import/extensions */
+/* eslint-disable jsx-a11y/accessible-emoji */
+import React from 'react';
+import * as S from '../styles';
+import { withErrorBoundary } from '../../../../ignitus-ErrorHandlingComponents/errorBoundary';
+import { AppIcon } from '../../../../types/iconsTypes/iconEnums';
+
+const ProfileCard = () => (
+  <S.ProfileDiv>
+    <S.ProfileContainer>
+      <S.ProfileTrack>
+        <S.BookmarkIconDiv>
+          <S.TextIconContent>Track Sophi&apos;s progress</S.TextIconContent>
+          <S.ProfileBookmarkIcon name={AppIcon.BookmarkIcon} />
+        </S.BookmarkIconDiv>
+      </S.ProfileTrack>
+      <S.ContentDiv>
+        <S.ImageContainer>
+          <img
+            src="https://storage.googleapis.com/ignitus_assets/ig-avatars/Team_avatar-eugene.png"
+            alt="ProfileLogo"
+          />
+        </S.ImageContainer>
+        <S.DetailsContainer>
+          <S.ProfileHeading>Sophia Carter</S.ProfileHeading>
+          <S.ProfileSubHeading>Literature Student</S.ProfileSubHeading>
+          <S.TextIconContainer>
```

Comment:
Create a component that will receive `content & icon & direction` as property.

Then reuse it wherever you have icons along with text.
In cases where you have icon before text direction property will come handy if you pass direction as reverse then  use flex-direction will become row-reverse.

To see how you can manipulate css by passing props in emotion refer here - https://github.com/Ignitus/Ignitus-client/blob/develop/src/ignitus-UserInterfaceBook/InterfaceSideNavigation/style.ts